src/shared-components/immersiveModal/style.ts

```diff
@@ -56,8 +63,13 @@ const CrossIconContainer = styled.button`
   cursor: pointer;
 
   ${MEDIA_QUERIES.mdUp} {
-    top: 16px;
+    top: ${({ showDesktopHeaderBar }) =>
+      showDesktopHeaderBar === true ? SPACER.x5large : '4.5rem'};
+    transition: top ${ANIMATION.defaultTiming}
```

Comment:
Since we're now using this as one button for desktop, I added a transition to make the positioning look sleeker with scrollable content:

https://user-images.githubusercontent.com/13544620/134046857-0ec2a925-feb7-419b-900d-4def6c421983.mov

The timing matches our other `opacity` transitions since they all rely on `ANIMATION.defaultTiming`.